tests/scripts/github-action-helper.sh

```diff
@@ -622,6 +622,23 @@ function test_csi_nfs_workload {
   kubectl exec -t pod/csinfs-demo-pod -- ls -alh /var/lib/www/html/
 }
 
+function install_minikube_with_none_driver() {
+  sudo apt update
+  sudo apt install -y conntrack socat
+  curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube_latest_amd64.deb
```

Comment:
I like the idea of just using the latest minikube release, but I also worry that it could break the CI unexpectedly, so we may want to stick with a specific version and update it occasionally, for example when K8s versions are updated.

Reply:
done